zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/storage/impl/FileSystemBackupStorage.java

```diff
@@ -44,19 +44,19 @@
 /**
  * Implementation for NFS-based backup storage provider
```

Comment:
Did you mean to update the documentation? Let's try to give some context here and how the user could extend it to various storage mounts (nfs, ssd, hdd, gpfs, etc.)

Reply:
Good catch. Thanks. Let me update the documentation.